Formula/osc-cli.rb

```diff
@@ -0,0 +1,38 @@
+class OscCli < Formula
+  include Language::Python::Virtualenv
+
+  desc "Official Outscale CLI providing connectors to Outscale API"
+  homepage "https://github.com/outscale/osc-cli"
+  url "https://files.pythonhosted.org/packages/8c/29/cf68365132a0ac906278abdc704569ce8f1ba4c609a2067cd00c5ed22271/osc-sdk-1.6.tar.gz"
+  sha256 "fb312c82c73ed4eba7bbb3a028e4d2f53e00e28c576ff2f638258f27a855e7e7"
+  license "BSD-3-Clause"
+
+  depends_on "python@3.9"
```

Comment:
```suggestion
  depends_on "python@3.9"
  depends_on "six"
```
Your test is failing because it's looking for this module.

Reply:
Specifically, the recursive PyPI dependencies are missing, so the formula is not going to work properly as various imports are going to fail.